components/compliance-service/integration_test/stats_server_read_summary_test.go

```diff
@@ -0,0 +1,314 @@
+
+			//nodes summary
+			expectedCompliantCnt:    1,
+			expectedNonCompliantCnt: 0,
+
+			//controls summary
+			expectedPassedCnt: 1,
+		},
+		{
+			description:     "Projects: user has access to unassigned reports",
+			allowedProjects: []string{authzConstants.UnassignedProjectID},
+
+			//report summary
+			expectedEnvironmentCnt: 1,
+			expectedPlatformCnt:    1,
+			expectedProfileCnt:     1,
+			expectedNodeCnt:        1,
+			expectedStatus:         "passed",
+
+			//nodes summary
+			expectedCompliantCnt:    1,
+			expectedNonCompliantCnt: 0,
+
+			//controls summary
+			expectedPassedCnt: 1,
+		},
+	}
+
+	for _, test := range successCases {
+		t.Run(test.description, func(t *testing.T) {
+			ctx := contextWithProjects(test.allowedProjects)
+
+			octoberTwentyFifthQuery := &stats.Query{
+				Filters: []*stats.ListFilter{
+					{Type: "end_time", Values: []string{"2018-10-25T23:59:59Z"}},
+				},
+			}
+			//passing in no type gets us a Summary type that contains a hydrated ReportSummary
+			response, err := statsServer.ReadSummary(ctx, octoberTwentyFifthQuery)
+
+			assert.NoError(t, err)
+			require.NotNil(t, response)
+
+			reportSummary := response.ReportSummary
+
+			//report summary
+			assert.Equal(t, test.expectedEnvironmentCnt, reportSummary.Stats.Environments, "Env count")
+			assert.Equal(t, test.expectedPlatformCnt, reportSummary.Stats.Platforms, "Platforms count")
+			assert.Equal(t, test.expectedProfileCnt, reportSummary.Stats.Profiles, "Profiles count")
+			assert.Equal(t, test.expectedNodeCnt, reportSummary.Stats.Nodes, "Nodes count")
+			assert.Equal(t, test.expectedStatus, reportSummary.Status, "status")
+
+			//passing in "nodes" type gets us a Summary type that contains a hydrated NodeSummary
+			octoberTwentyFifthQuery.Type = "nodes"
+			response, err = statsServer.ReadSummary(ctx, octoberTwentyFifthQuery)
+
+			assert.NoError(t, err)
+			require.NotNil(t, response)
+
+			nodeSummary := response.NodeSummary
+
+			//node summary
+			assert.Equal(t, test.expectedCompliantCnt, nodeSummary.Compliant, "Compliant count")
+			assert.Equal(t, test.expectedNonCompliantCnt, nodeSummary.Noncompliant, "NonCompliant count")
+			assert.Equal(t, test.expectedHighRiskCnt, nodeSummary.HighRisk, "HighRisk count")
+			assert.Equal(t, test.expectedLowRiskCnt, nodeSummary.LowRisk, "LowRisk count")
+			assert.Equal(t, test.expectedMediumRiskCnt, nodeSummary.MediumRisk, "MediumRisk count")
+			assert.Equal(t, test.expectedSkippedCnt, nodeSummary.Skipped, "Skipped count")
+
+			//passing in "controls" type gets us a Summary type that contains a hydrated ControlSummary
+			octoberTwentyFifthQuery.Type = "controls"
+			response, err = statsServer.ReadSummary(ctx, octoberTwentyFifthQuery)
+
+			assert.NoError(t, err)
+			require.NotNil(t, response)
+
+			controlSummary := response.ControlsSummary
+
+			//control summary
+			assert.Equal(t, test.expectedPassedCnt, controlSummary.Passed, "Passed count")
+		})
+	}
```

Comment:
I'd make the same comment as on the other PR: https://github.com/chef/automate/pull/322#discussion_r283510492

Reply:
making that change now.. thanks